docs/flags.md

```diff
@@ -13,7 +13,7 @@ $ mycli --force --file=./myfile
 It would be declared like this:
 
 ```js
-import Command, {flags} from '@oclif/command'
+import {Command, flags} from '@oclif/command'
```

Comment:
`Command` is the default export of @oclif/command.

Reply:
Okay got it 👍